### PR TITLE
Add remote seat heater requests

### PR DESCRIPTION
**Pull request by jor-dan**
Add documentation for creating a remote seat heater request and update the response that climate state data requests return.

### CODE REVIEW

File: docs/vehicle/commands/climate.md [hunk 1]
```diff
@@ -47,3 +47,23 @@ Note: The parameters are always in celcius, regardless of the region the car is
...
+
+## POST `/api/1/vehicles/{id}/command/remote_seat_heater_request`
+
+Sets the specified seat's heater level.
+
+### Parameters
+
+| Parameter | Example | Description                             |
+| :-------- | :------ | :-------------------------------------- |
+| heater    | 0       | The desired seat to set the heater for. |
```
Comment (timdorr) [inline docs/vehicle/commands/climate.md]: Do these numbers correspond with a particular seat? We should probably document that.

Comment (jor-dan) [reply docs/vehicle/commands/climate.md]: Yes. This is how the numbers map to seats in a 5 seat configuration:

| Number | Seat            | ClimateState seat heater name |
| :------- | :----------- | :------------------------------ |
| 0            | Driver          | seat_heater_left                          |
| 1            | Passenger   | seat_heater_right                        |
| 2            | Rear left      | seat_heater_rear_left                  |
| 3            | Rear center | seat_heater_rear_center            |
| 4            | Rear right    | seat_heater_rear_right               |

**Comment by timdorr** [comment]
Thanks!

File: docs/vehicle/commands/climate.md (hunk 1 above)
Comment (jor-dan) [reply docs/vehicle/commands/climate.md]: Oops. I double checked and realized the values for the rear center and rear right seats are off by 1. This should be the correct mapping:

| Number | Seat            |
| :------- | :----------- |
| 0            | Driver          |
| 1            | Passenger   |
| 2            | Rear left      |
| 4            | Rear center |
| 5            | Rear right    |

A value of 3 doesn't seem to do anything on a 5 seat configuration. I don't think Tesla would randomly skip 3 so it probably works on 6 or 7 seat configurations. My bad!

Comment (timdorr) [reply docs/vehicle/commands/climate.md]: Oops, I'll fix that. 

BTW, it's probably a Monty Python reference: https://youtu.be/xOrgLj9lOwk?t=129

**Pull request merged**